kani-compiler/src/codegen_cprover_gotoc/codegen/block.rs

```diff
@@ -1,12 +1,16 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-//! This file contains functions related to codegenning MIR blocks into gotoc
```

Comment:
Why remove this? I feel like having at least one comment is better than not having it at all.

Reply:
I basically just moved the comment from the file to the function.

Reply:
Right, I thought this implementation was way longer.